myapp.rb

```diff
@@ -4,5 +4,17 @@
 require "sinatra/reloader" if development?
 
 get '/' do
-  'Hello world!'
+  erb :home
+end
+
+get '/aboutme' do
```

Comment:
```suggestion
get '/about_me' do
```

use `_` para separar